packages/orchestration/tools/ibc-mocks.ts

```diff
@@ -207,12 +209,19 @@ export const buildVTransferEvent = ({
   sourceChannel = 'channel-405' as IBCChannelID,
   sequence = 0n,
   memo = '',
+  acknowledgementError,
 }: BuildVTransferEventParams = {}): VTransferIBCEvent => ({
   type: VTRANSFER_IBC_EVENT,
   blockHeight: 0,
   blockTime: 0,
   event,
-  acknowledgement: btoa(JSON.stringify({ result: 'AQ==' })),
+  acknowledgement: btoa(
+    JSON.stringify(
+      acknowledgementError
+        ? { error: acknowledgementError }
+        : { result: 'AQ==' },
+    ),
+  ),
```

Comment:
I owe a test for this - probably in `local-orchestration.account.test.ts`. Looking for an error from MN but i think it was longer than our log retention window

Reply:
```
2025-03-13T06:19:55.427Z SwingSet: ls: v221 Error#3: ICS20-1 transfer error packet-forward-middleware error: giving up on packet on channel (channel-21) port (transfer) after max retries
```

Reply:
[e80ccd9](https://github.com/Agoric/agoric-sdk/pull/11318/commits/e80ccd9dd252d518636142e07789846d250f9b4c)